server/app.js

```diff
@@ -19,6 +20,14 @@ app.use(express.urlencoded({ extended: false }));
 
 app.use('/api/v1/', routes);
 
+app.use(express.static(join(__dirname, '..', 'client', 'build')));
+
+if (process.env.NODE_ENV === 'production') {
+  app.get('*', (req, res) => {
+    res.sendFile(join(__dirname, '..', 'client', 'build', 'index.html'));
```

Comment:
There is no build folder in client folder , Does you mean bublic instead of build ?

Reply:
Because this folder will be created when you build your app and in `production` we run the `build` script

Reply:
![Screenshot from 2021-02-02 11-00-49](https://user-images.githubusercontent.com/42492094/106576570-f99e9300-6545-11eb-80c5-17e340da32f3.png)

Reply:
When we are in the production stage we should build the react app, and that will create a build file.